test/unit_Polygon.jl

```diff
@@ -106,8 +106,26 @@ for N in [Float64, Float32] # TODO Rational{Int}
     @test ⊆(p2, p1) && ⊆(p2, p1, true)[1]
     @test ⊆(HPolygon{N}(), p1)
     @test ⊆(p1, BallInf(N[1., 0.], N(1.)))
+
+    v1 = N[1.0, 0.0]
+    v2 = N[1.0, 1.0]
+    v3 = N[0.0, 1.0]
+    v4 = N[-1.0, 1.0]
+    v5 = N[-1.0, 0.0]
+    v6 = N[-1.0, -1.0]
+    v7 = N[0, -1.0]
+    v8 = N[1.0, -1.0]
+    v = [v1, v2, v3, v4, v5, v6, v7, v8]
+
+    for (i, vi) in enumerate(v)
+        for j in i:8
```

Comment:
`1:i`, right? (same below)

Reply:
for these definitions of `vi`, we should check that `v1 <= v1`, `v1 <= v2`, `v1 <= v3`, and so on until `v1 <= v8`. similarly `v2 <= v2`, until `v2 <= v8`. `v8` is the greatest angle thus it is only tested against itself.

Reply:
Oh, I read `i` as `1`, is see now what you did.